ci-operator/config/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-main.yaml

```diff
@@ -68,9 +68,9 @@ tests:
   commands: openshift/unit-tests.sh
   container:
     from: src
-- as: e2e-aws-capi-techpreview
+- always_run: false
```

Comment:
@damdo @JoelSpeed Should we promote our CAPI TechPreview jobs from optional to required? Given that we want at least AWS to GA soon, I think it would be valuable to enforce them, at least there.